### ISSUE TITLE
Asymmetric Encryption System

### ISSUE THREAD

**Issue by rexdivakar** [Open-Dev-Community/QrCode-OTP#1]
Create a asymmetric RSA encryption model to encrypt the database

**Comment by TheYk98** [comment]
hey do you just want a generic Rsa encrypter ? I didnt get the issue

**Comment by rexdivakar** [comment]
yeah we need a rsa encryption system to generate keys rest i will manage @TheYk98

**Comment by TheYk98** [comment]
cool. lemme try this out

**Comment by rexdivakar** [comment]
Issue assigned to @TheYk98 
🌟 Star it 🍴Fork it 🤝 Contribute to it!
Happy Coding..

Feel free to drop in https://discord.gg/59TDJgt

**Comment by TheYk98** [comment]
hey sry was a little busy. will start off this weekend
